tests/transforms/test_transform_pipeline.py

```diff
@@ -130,6 +137,6 @@ def test_transform_pipeline(tmp_path, monkeypatch, mock_db_conn_fixture,
         mock_imageio.imsave.assert_has_calls(calls, any_order=False)
 
     # Assert that created manifests are correct
-    outdir_name = f"segmentation_run_id_{input_data['segmentation_run_id']}"
+    outdir_name = f"seg_run_id_{input_data['segmentation_run_id']}"
     expected_outdir = Path(input_data["artifact_basedir"]) / outdir_name
-    expected_calls = [call(m, expected_outdir) for m in expected_manifests]
+    [call(m, expected_outdir) for m in expected_manifests]
```

Comment:
Why was `expected_calls = [call(m, expected_outdir) for m in expected_manifests]` changed to `[call(m, expected_outdir) for m in expected_manifests]`? The line now doesn't do anything?

Reply:
flake8 did not like that the variable `expected_calls` was assigned to and never used.
The line still executes `call()`:
```
In [29]: def myfun(i): 
    ...:     print(i) 
    ...:                                                                           

In [30]: [myfun(i) for i in range(4)]                                              
0
1
2
3
```

Reply:
That `call()` is a helper for python mocking (https://docs.python.org/3/library/unittest.mock.html#call). So this line is saying setting up the expectation that something will have been called with the `expected_manifests` and the `expected_outdir`. There is a line missing then where `assert_has_calls` for a magic mock object is called.

Reply:
It looks like what is missing is:

`mock_db_conn_fixture.bulk_insert.assert_has_calls(expected_calls)`

Reply:
odd, you did have this, and it looks like I removed it here: https://github.com/AllenInstitute/segmentation-labeling-app/pull/37/commits/86cfe430a27158615451f688f0ed77e24d99a25a as part of https://github.com/AllenInstitute/segmentation-labeling-app/pull/37
I'm not sure why, I will restore

Reply:
Probably because I was literally saving a json output manifest there which has been replaced with saving to a database.

Reply:
oh, of course. thanks for refreshing my memory

Reply:
added an `assert_has_calls` on `bulk_insert`